src/main/java/cz/muni/fi/pv286/mmt/model/FromOrTo.java

```diff
@@ -0,0 +1,9 @@
+package cz.muni.fi.pv286.mmt.model;
+
+/**
+ * Enum for From or To input parameters.
+ */
+public enum FromOrTo {
```

Comment:
**answer:** From

**issue:** Naming

It just look funny in code: FromOrTo.FROM 😄